lib/config/default.js

```diff
@@ -413,6 +413,17 @@ defaultConfig.definition = () => ({
       default: false
     }
   },
+  cloud: {
+    aws: {
+      /**
+       * The AWS account ID for the AWS account associated with this app.
+       */
+      account_id: {
+        default: null,
+        env: 'NEW_RELIC_CLOUD_AWS_ACCOUNT_ID'
```

Comment:
not needed and i would add unit tests in `config-env` and `config-defaults` for this new one. also make sure you PR the docs-website for this new option. Also this needs a formatter of `int`

Reply:
Done

Reply:
I will pr the docs website after the dynamo ticket is completed because then we will actually be using this account id.